### ISSUE TITLE
connected users

### ISSUE THREAD

**Issue by peekaa** [OpenMediaVault-Plugin-Developers/openmediavault-autoshutdown#26]
Even if I have CHECK_CLI="true", I am logged to webgui administration as admin, I am logged in shellinabox ssh as root, still autoshutdowm evaluates this as false and shut down the system. Is it bug or my misunderstanding of "Check for connected users."? Thank you.

**Comment by ryecoaaron** [comment]
The script uses the w command to see who is logged in to the system.  When you login via the webgui, you aren't really logging into the system.  Not sure why shellinabox doesn't register.  From shellinabox, what is the output of:
w -h

**Comment by peekaa** [comment]
```
root@box:~# w -h                                                                                                                      
root     pts/1    localhost        19:58    2.00s  0.08s  0.00s w -h         
```

**Comment by ryecoaaron** [comment]
Actually I need the output of:
w -h | egrep -v '^\s*$'

**Comment by peekaa** [comment]
```
root@box:~# w -h | egrep -v '^\s*$'                                                                                                   
root     pts/1    localhost        20:12    7.00s  0.07s  0.00s w -h           
```                                                       
Its the same. It seems to me, it could be connected with locales. When I am now switched to english, in system log there are no info at all, about what is going on, just this: even if I have Verbose mode....
```
Dec 30 20:11:20 box systemd[1]: Starting autoshutdown...
Dec 30 20:11:20 box systemd[1]: Started autoshutdown.
```

**Comment by ryecoaaron** [comment]
I don't know why it isn't working with shellinabox then.  Unfortunately, I didn't write this script and I don't use it.  So, it is difficult for me to make changes especially to accommodate locales.

**Comment by peekaa** [comment]
OK, thank you anyway.

**Comment by nrandon** [comment]
corrected in 5.1.3
